code/datums/ruins/space.dm

```diff
@@ -148,6 +148,7 @@
 	suffix = "spacehotel.dmm"
 	name = "The Twin-Nexus Hotel"
 	description = "An interstellar hotel, where the weary spaceman can rest their head and relax, assured that the residental staff will not murder them in their sleep. Probably."
+	unpickable = TRUE	//Splurt Change
```

Comment:
could just add this modularly tbh

Reply:
Not sure how to go about that in this case - could you suggest a method?

Reply:
> Not sure how to go about that in this case - could you suggest a method?

create a modular_splurt/code/datums/ruins/space.dm (if there isn't one already) and add something like

```
/datum/map_template/ruin/space/spacehotel
	unpickable = TRUE
```